src/styles/atoms/_atoms.table.scss

```diff
@@ -53,6 +53,10 @@
     &.is-clickable:focus {
       background-color: var(--table-highlight-bg-color);
       cursor: pointer;
+
+      .small {
+        color: $gray-500;
```

Comment:
moet dit niet ook een CSS custom property zijn?

Reply:
Ik twijfel daar wat over. Het is eigenlijk een contrast fix die enkel hier van toepassing is. Moeten we dat dan overschrijfbaar maken?